ansible/site.yml

```diff
@@ -1,11 +1,4 @@
 ---
-# This playbook stands up client test systems in AWS and then uses them to 
-# stress test a FHIR server.
-#
-# Usage:
-#  
-#     $ ansible-playbook site.yml --extra-vars "ec2_key_name=foo num_slaves=10 fhir_server=http://example.com/baseDstu2"
-
 - hosts: localhost
   name: AWS - Provision Resources and Instances
```

Comment:
You should fix the name of this play and remove the commented-out code.

Reply:
Renamed palyabook to "fhir-stress-test.yml" and removed usage comment.  Updated README.md instead.